probes/releasesHaveVerifiedProvenance/impl.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//nolint:stylecheck
+package releasesHaveVerifiedProvenance
+
+import (
+	"embed"
+	"fmt"
+
+	"github.com/ossf/scorecard/v5/checker"
+	"github.com/ossf/scorecard/v5/finding"
+	"github.com/ossf/scorecard/v5/internal/probes"
+)
+
+func init() {
+	probes.MustRegister(Probe, Run, []probes.CheckName{probes.SignedReleases})
+}
+
+//go:embed *.yml
+var fs embed.FS
+
+const (
+	Probe = "releasesHaveVerifiedProvenance"
+)
+
+func Run(raw *checker.RawResults) ([]finding.Finding, string, error) {
+	var findings []finding.Finding
+
+	if len(raw.SignedReleasesResults.Packages) == 0 {
+		f, err := finding.NewNotAvailable(fs, Probe, "no package manager releases found", nil)
```

Comment:
traditionally we've used `NotApplicable` for cases where there are no X (where X is workflows, releases, commits, etc.)